source/core/slang-token-reader.cpp

```diff
@@ -681,9 +681,6 @@ namespace Misc {
         {
             switch (data[i])
             {
-            case ' ':
```

Comment:
There are more complete versions of this functionality in StringEscapeUtil. Not directly an issue for PR, but would be nice to have have DRY.